src/write_gcm.c

```diff
@@ -4183,6 +4193,10 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
           metadata);
     wg_payload_destroy(to_remove);
   }
+  if (drop_count != 0) {
```

Comment:
How do we know that the queue has recovered at this point? Or is the intent that, each time we try to write, we tell you how many points we dropped for that write? If so, then I'd be inclined to suggest we remove "Queue recovered successfully" because we don't really know if it has recovered - the same issue might exist the next time points are written.